bots/jabber_mirror_backend.py

```diff
@@ -90,15 +96,18 @@ def __init__(self, jid, password, rooms):
         self.register_plugin('xep_0199') # XMPP Ping
 
     def set_zulip_client(self, client):
+        # type: (zulip.Client) -> None
         self.zulip = client
 
     def session_start(self, event):
+        # type: (Any) -> None
```

Comment:
Well, yeah, but I think that the fact that the `event` argument is not used anywhere in the function should be taken into account. Since `event` could safely be removed from the function definition, literally anything can be passed as an argument.